index.js

```diff
@@ -398,8 +398,10 @@ InfluxDB.prototype._prepareValues = function (series) {
         if (timestamp) {
           if (timestamp instanceof Date) {
             line += ' ' + timestamp.getTime()
+          } else if (/^[0-9]+$/.test(timestamp)) {
+            line += ' ' + timestamp  // UNIX timestamp
           } else {
-            line += ' ' + timestamp
+            line += ' ' + new Date(timestamp).getTime()  // hopefully an RFC3339 string
```

Comment:
This will misbehave if the `timePrecision` option is manually or by default set to anything other than `ms`. We should, perhaps, parse into Influx's time format when possible to ensure that nothing goes wrong here. You could copy the formatting function I have on the 5.0.0 branch [here](https://github.com/node-influx/node-influx/blob/5.0.0/src/grammar.ts#L101).